dune-project

```diff
@@ -1,3 +1,3 @@
-(lang dune 2.0)
+(lang dune 2.2)
```

Comment:
this implies bumping the dependency to dune to `>= 2.2` in `mdx.opam`. I'll defer to @NathanReb to decide if it's OK. On a related note, mdx does not use opam file generation and this would detect this inconsistency. we can set this up if you're interested (in a separate PR)

Reply:
I bumped the constraint in Opam as well, so if we think it's ok to increase the lower bound, this is solved.
I'll send a PR to generate the opam file automatically.

Reply:
Yeah bumpin minor dune versions is :+1:

Reply:
PR pushed in #302